projects/views.py

```diff
@@ -1,13 +1,22 @@
 from django.shortcuts import render, get_object_or_404
+from django.http import HttpResponse, HttpResponseRedirect
+from django.template import loader
 from .models import Post
 import random
 
 
 # Create your views here.
 def home(request):
-    print('Hello ')
-    posts = Post.objects
-    return render(request, 'home.html', {'posts': posts})
+    posts = Post.objects.all()
+    rand_post = random.choice(posts)
+    content_post = Post.content
```

Comment:
this Post model manager is declared but not used anywhere in the content.
I dont think we need this manager to display our content. 
we only need Post querieys such as
`Post.objects.all()`  
or `Post.objects.filter()`  to get a filtered posts list
 or `Post.objects.get()` get a specific post